UNIXFS.md

```diff
@@ -61,12 +61,25 @@ message Data {
 	optional uint64 hashType = 5;
 	optional uint64 fanout = 6;
 	optional uint32 mode = 7;
-	optional int64 mtime = 8;
+	optional TimSpec mtime = 8;
 }
 
 message Metadata {
 	optional string MimeType = 1;
 }
+
+message TimeSpec {
+	// Use of int64 is deliberate - negative epoch is super unlikely
```

Comment:
This sort of discussion should move to the `Design decision rationale` section

Reply:
Fixing...